doc/conf.py

```diff
@@ -58,13 +58,6 @@
 # The master toctree document.
 master_doc = "index"
 
-# The language for content autogenerated by Sphinx. Refer to documentation
-# for a list of supported languages.
-#
-# This is also used if you do content translation via gettext catalogs.
-# Usually you set "language" from the command line for these cases.
-language = None
```

Comment:
This is no longer recommended in Sphinx.  It will default to English if no language is specified.